runtime/onert/api/nnfw/include/nnfw_experimental.h

```diff
@@ -620,24 +620,33 @@ NNFW_STATUS nnfw_odc_delete_minmax_file(nnfw_session *session);
 /**
  * @brief  Run inference with auto compilation
  *
- * <p>This function runs inference with automatic compilation and replaces
- *  the original model with a quantized or compiled model inside.
- * During the inference the minmax statistics is collected and after that quantization is performed.
- * If quantization was successful, try to code generating for target backend, otherwise run original
- float model.
+ * This function runs inference float model with automatic compilation and
+ * replaces the original model with a quantized or compiled model inside.
+ *
+ * During the inference, the minmax statistics is collected and after that
+ * quantization is performed.
+ * If quantization was successful, try to code generating for target backend,
+ * otherwise run original float model.
+ *
  * If compilation was successful, run compiled model, otherwise run quantized model.
- * On-device compiler (ODC) provides quantization and compilation functionality.
- * Function should be called after model is loaded by {@link nnfw_load_model_from_file},
- * session is prepared for inference by {@link nnfw_prepare}, set input and output buffers
- * by {@link nnfw_set_input} and {@link nnfw_set_output}.
+ *
+ * Auto compilation uses on-device compiler (ODC), and ODC provides
+ * quantization and compilation functionality.
  *
  * Additionally the following parameters should be set up :
  * 1. Quantization type {@link nnfw_set_quantization_type }
  * 2. Quantizated model path {@link  nnfw_set_quantized_model_path }
```

Comment:
(Optional)
Could you fix this typo as well?
```suggestion
 * 2. Quantized model path {@link  nnfw_set_quantized_model_path }
```